items/GHAST_MONSTER.json

```diff
@@ -67,25 +45,25 @@
         },
         {
           "id": "DYE_CYCLAMEN",
-          "chance": "0.00004%"
+          "chance": "0.00008333%"
```

Comment:
Where does this come from? (Same with other Cyclamen dye chances i dont want to comment it on every single piece)

Reply:
reverted, sorry i wasn't aware hypixel changed the drop chances in bestiary if the dye was boosted